docs/Guidelines(8th Edition).xml

```diff
@@ -0,0 +1,251 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+	<guideline key="Ch01_fa67753" severity="DO" section="Naming" subsection="Variables and fields">DO favor clarity over brevity when naming identifiers.</guideline>
+	<guideline key="Ch01_674fe1e" severity="DO NOT" section="Naming" subsection="Abbreviations">DO NOT use abbreviations or contractions within identifier names.</guideline>
+	<guideline key="Ch01_024aa0c" severity="UNKNOWN ARGL" section="Naming" subsection="Abbreviations">"DO NOT use any acronyms unless they are widely accepted, and even then use them consistently."</guideline>
```

Comment:
It looks like the severity on several of these items is set to UNKNOWN ARGL. From looking over the items it appears that it is likely caused by the leading quotation marks

Reply:
@Keboo oh you are right, with the quotes the automatic recognition of the severtiy didn't work correctly. I will fix this.

Reply:
@Keboo done